### DISCUSSION TITLE
Spoken language dependencies

### DISCUSSION THREAD

**Discussion by bitcarousel** [pyannote/pyannote-audio#610] [Q&A]
Am I correct in assuming that the speaker diarization pipeline (sad, scd, emb) has little dependence on the spoken language.
Reason for this statement is that MFCC as well as SincNet represent the signal in the frequency domain which probably has less dependencies to the spoken language?
The advantage of training (e.g. finetune) the models is more to deliver improved results on the **own** data independent of the spoken language. 
Can I assume that this statement true?

**Comment by hbredin** [comment]
As with any machine learning problem, it is always better to train models with data following the same distribution as that of the data used during test.

That being said, it is true that speaker diarization is less dependent on the spoken language than other tasks such as automatic speech recognition (ASR).
